crates/virtio-queue/src/mock.rs

```diff
@@ -407,7 +477,9 @@ impl<'a, M: GuestMemory> MockSplitQueue<'a, M> {
             };
 
             let desc = Descriptor::new(addr, len, flags, next);
-            self.desc_table().store(i, desc);
+            if let Err(e) = self.desc_table().store(i, desc) {
```

Comment:
This can be written as: ```self.desc_table().store(i, desc)?;```. Same for all the other similar code patterns in this PR.

Reply:
Done